### PR TITLE
add binance paper trade endpoint

### PR DESCRIPTION
**Pull request by zenixls2**
options:
1. use environment variable
2. define it in config yaml

also, what is the influence of our db data once this option is enabled?  
anyway to separate them?

### CODE REVIEW

**Comment by zenixls2** [comment]
choose to support this feature from environment variable.
ready to be reviewed.

File: README.md [hunk 1]
```diff
@@ -173,6 +173,15 @@ bbgo pnl --exchange binance --asset BTC --since "2019-01-01"
 
 ## Advanced Configuration
 
+### Testnet (Paper Trading)
+
+Currently only supports binance testnet.
+To run bbgo in testnet, apply new API keys from [Binance Test Network](https://testnet.binance.vision), and set the following env before you start bbgo:
+```bash
+export PAPER_TRADE=1
```
Comment (c9s) [inline README.md:181]: Add `BINANCE_` as the prefix?

File: pkg/service/sync.go [hunk 2]
```diff
@@ -44,27 +51,28 @@ func (s *SyncService) SyncSessionSymbols(ctx context.Context, exchange types.Exc
 		}
 	}
 
-	log.Infof("syncing %s deposit records...", exchange.Name())
-	if err := s.DepositService.Sync(ctx, exchange); err != nil {
-		if err != ErrNotImplemented {
-			return err
+	if !paperTrade() {
```
Comment (c9s) [inline pkg/service/sync.go:54]: early return?

```
if paperTrade() {
    return nil
}
```

File: pkg/service/sync.go [hunk 3]
```diff
@@ -23,6 +25,11 @@ type SyncService struct {
 	DepositService  *DepositService
 }
 
+func paperTrade() bool {
+	v, err := strconv.ParseBool(os.Getenv("PAPER_TRADE"))
```
Comment (c9s) [inline pkg/service/sync.go:29]: you can use something like this:

```
if b, ok := util.GetEnvVarBool("DEBUG_SLACK"); ok { ... }
```

File: README.md (hunk 1 above)
Comment (zenixls2) [reply README.md:181]: okex also has paper trade endpoints. keep this to avoid additional modification.

File: pkg/service/sync.go (hunk 3 above)
Comment (zenixls2) [reply pkg/service/sync.go]: done

File: pkg/service/sync.go (hunk 2 above)
Comment (zenixls2) [reply pkg/service/sync.go]: done

**Review by c9s — APPROVED** [review]

**Pull request merged**